FixMyBerlin/fixmy.frontend
# Fix intersection popup

- Add back intersection popups for the HBI map
- Add an (unrelated) e2e test for the detail panel in the projects map
- Fix test 'closes the popup on close button click', which would fail if the close button was covered by another element
- Add very simple unit test for `WebglMap`
- Update module structure so that `WebglMap` can be imported as a named import

Closes #592 

## How Has This Been Tested?

I did not add unit tests to this one because that would require some setup code that retrieves the coordinates of an intersection to click. The `WebglMap` doesn't provide an interface that would allow this from a Jest test. So instead it's just manual testing here.

- Visit the HBI map at /zustand
- Click an intersection
- Observe a new popup that informs you about missing information for intersections

## Checklist:
